src/agent/command_handler/src/command_handler.cpp

```diff
@@ -5,7 +5,8 @@ namespace command_handler
     const std::unordered_map<std::string, std::pair<std::string, module_command::CommandExecutionMode>>
         VALID_COMMANDS_MAP = {
             {"set-group", {"CentralizedConfiguration", module_command::CommandExecutionMode::SYNC}},
-            {"update-group", {"CentralizedConfiguration", module_command::CommandExecutionMode::SYNC}}};
+            {"update-group", {"CentralizedConfiguration", module_command::CommandExecutionMode::SYNC}},
+            {"restart", {"restart", module_command::CommandExecutionMode::SYNC}}};
```

Comment:
After rebase, add command name [here](https://github.com/wazuh/wazuh-agent/blob/master/src/agent/command_handler/include/command_entry/command_entry.hpp#L10). (also module name)

Reply:
Done. Thanks for the feedback and the pointer!